roles/k8s_best_practices_certsuite/defaults/main.yml

```diff
@@ -25,7 +25,7 @@ kbpc_image_suffix: "{{ lookup('community.general.random_string', length=16, spec
 # Provide feedback for each certsuite test launched. Just copy this template and fill the value for the
 # tests you want to provide feedback. You don't need to include the tests that you don't want to provide feedback.
 # To insert line breaks, use \\n (you need to escape it). Same for quotes and so on.
-# List of tests updated up to kbpc_version
+# List of tests updated up to kbpc_version = v5.3.2
```

Comment:
I think we can skip changing this every new release by simply referencing the version set in the `kbpc_version` variable

Reply:
ack